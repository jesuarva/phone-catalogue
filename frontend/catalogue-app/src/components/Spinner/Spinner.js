import React from 'react';
import SPINNER from '../../media/rotating-balls-spinner.gif';

const Spinner = () => {
  return (
    <figure className="figure">
      <img
        src={SPINNER}
        alt="masmovil logo"
        className="figure-img img-fluid rounded"
      />
    </figure>
  );
};

export default Spinner;
