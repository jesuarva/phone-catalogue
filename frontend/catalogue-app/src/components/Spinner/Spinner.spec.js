import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './Spinner';

configure({ adapter: new Adapter() });

describe('Spinner component', () => {
  const spinnerComponent = <Spinner />;

  it('should renders correctly', () => {
    const tree = renderer.create(spinnerComponent).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
