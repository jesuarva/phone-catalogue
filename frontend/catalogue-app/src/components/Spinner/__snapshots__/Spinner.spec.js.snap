// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component should renders correctly 1`] = `
<div
  className="col-12 custom-card"
>
  <img
    alt="masmovil logo"
    className="figure-img img-fluid rounded"
    src="rotating-balls-spinner.gif"
  />
</div>
`;
